spyder_unittest/unittestplugin.py

```diff
@@ -148,6 +148,14 @@ def on_projects_available(self):
         projects.sig_project_loaded.connect(self.handle_project_change)
         projects.sig_project_closed.connect(self.handle_project_change)
 
+    @on_plugin_available(plugin=Plugins.PythonpathManager)
+    def on_pythonpath_manager_available(self):
+        """
+        Connect to signal announcing that Python path changed.
+        """
+        pythonpathManager = self.get_plugin(Plugins.PythonpathManager)
+        pythonpathManager.sig_pythonpath_changed.connect(self.update_pythonpath)
```

Comment:
```suggestion
        ppm = self.get_plugin(Plugins.PythonpathManager)
        ppm.sig_pythonpath_changed.connect(self.update_pythonpath)
```

This is the acronym we decided to use for the Pythonpath manager in Spyder.